app/views/content/teacher-training-advisers/_blog_links.html.erb

```diff
@@ -0,0 +1,27 @@
+  <section>
```

Comment:
I agree that these blocks work as is @EllieNodder - would suggest taking them out for now until @gazcarr has thought about a new component